src/test/features/claim/routes/defendant-change.ts

```diff
@@ -0,0 +1,121 @@
+import { expect } from 'chai'
+import * as request from 'supertest'
+import * as config from 'config'
+import * as mock from 'nock'
+
+import '../../../routes/expectations'
+import { checkAuthorizationGuards } from './checks/authorization-check'
+
+import { Paths as ClaimPaths } from 'claim/paths'
+
+import { app } from '../../../../main/app'
+
+import * as idamServiceMock from '../../../http-mocks/idam'
+import * as draftStoreServiceMock from '../../../http-mocks/draft-store'
+
+const cookieName: string = config.get<string>('session.cookieName')
+const roles: string[] = ['solicitor']
+
+const defendants = {
+  defendants: [{
+    address: {
+      line1: 'Apt 99',
+      city: 'London',
+      postcode: 'E1'
+    },
+    representative: {
+      organisationName: 'Defendant Organisation Name',
+      address: {
+        line1: 'Apt 99',
+        line2: 'Building A',
+        city: 'London',
+        postcode: 'E1'
+      }
+    },
+    defendantDetails: {
+      type: 'INDIVIDUAL',
+      title: 'title',
+      fullName: 'fullName'
+    }
+  },
+    {
+      address: {
+        line1: 'Apt 99',
+        city: 'London',
+        postcode: 'E1'
+      },
+      representative: {
+        organisationName: 'Defendant Organisation Name',
+        address: {
+          line1: 'Apt 99',
+          line2: 'Building A',
+          city: 'London',
+          postcode: 'E1'
+        }
+      },
+      defendantDetails: {
+        type: 'INDIVIDUAL',
+        title: 'title',
+        fullName: 'fullName'
+      }
+    }]
+}
+
+describe('Claim issue: Defendant change page', () => {
+  beforeEach(() => {
+    mock.cleanAll()
+    draftStoreServiceMock.resolveSave('legalClaim')
+    draftStoreServiceMock.resolveRetrieve('view')
+    draftStoreServiceMock.resolveSave('view')
+    idamServiceMock.resolveRetrieveUserFor(1, ...roles)
+  })
+
+  describe('on GET', () => {
+    checkAuthorizationGuards(app, 'get', ClaimPaths.defendantChangePage.uri)
+
```

Comment:
all test message just talks about redirecting page, but why is buried in post request, shouldn't a word saying if update is for address, representative or type